### PR TITLE
Add map styles for national

### PR DESCRIPTION
**Pull request by tinashechiraya**
[Screencast from 22-08-23 20:16:38.webm](https://github.com/kartoza/sawps/assets/70011086/76eccdd8-666e-42ec-bd22-d75cb42e60c3)

### CODE REVIEW

**Comment by tinashechiraya** [comment]
Description:

Map styles for national level
1. when on national level/decision maker role ,as demonstrated in the video user cant zoom in to deeper levels like viewing properties property etc

File: django_project/frontend/utils/country_level_dark_v1.json [hunk 1]
```diff
@@ -0,0 +1,583 @@
...
+      "tiles": [
+        "http://aerial.openstreetmap.org.za/ngi-aerial/{z}/{x}/{y}.jpg"
+      ],
+      "minzoom": 0,
+      "maxzoom": 24,
+      "layout": {"visibility": "none"}
+    }
+  },
+  "sprite": "",
+  "glyphs": "https://api.maptiler.com/fonts/{fontstack}/{range}.pbf?key={key}",
```
Comment (danangmassandy) [inline django_project/frontend/utils/country_level_dark_v1.json:32]: please use `key={{MAPTILER_API_KEY}}`

File: django_project/frontend/utils/country_level_dark_v1.json [hunk 2]
```diff
@@ -0,0 +1,583 @@
+{
+  "version": 8,
+  "name": "sanbi",
+  "metadata": {
+    "maputnik:license": "https://github.com/maputnik/osm-liberty/blob/gh-pages/LICENSE.md",
+    "maputnik:renderer": "mbgljs",
+    "openmaptiles:version": "3.x",
+    "sanbi:version": "12",
```
Comment (danangmassandy) [inline django_project/frontend/utils/country_level_dark_v1.json:8]: Use version sanbi:version 1 for this style.

File: django_project/frontend/utils/country_level_dark_v1.json [hunk 3]
```diff
@@ -0,0 +1,583 @@
...
+    "sanbi:theme": "dark"
+  },
+  "center": [25.86, -28.52],
+  "zoom": 5,
+  "sources": {
+    "sanbi": {
+      "type": "vector",
+      "tiles": ["http://localhost:9090/maps/sanbi/{z}/{x}/{y}.pbf"],
+      "minZoom": 0,
```
Comment (danangmassandy) [inline django_project/frontend/utils/country_level_dark_v1.json:17]: I think let's set to minZoom: 5. This is one of the change that I need to do for importing Jeremy style.
Can also change other occurrance for minZoom: 0?
Please also apply to light theme

**Review by danangmassandy — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 26 min

**Review by danangmassandy — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 25 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 27 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 26 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 29 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 27 min

**Comment by tinashechiraya** [comment]
national_level_map
on staging site the appearance of nation level map

**Pull request merged**